_custombuild

```diff
@@ -53,17 +53,13 @@ var components = gulp.src([
   APP_ROOT + 'components/gaia-dialog/gaia-dialog.js',
```

Comment:
We should do an audit of these at some point and make sure that we actually need them all.